test/fixtures/transport-main.js

```diff
@@ -0,0 +1,9 @@
+'use strict'
+
+const { join } = require('path')
+const pino = require('../..')
+const transport = pino.transport({
+  target: join(__dirname, 'transport-worker.js')
+})
+const logger = pino(transport)
+logger.info('Hello')
```

Comment:
is it intentional for fixture to export nothing and do something on import?

Reply:
yes, this is called by a child process test.

Reply:
I'm mostly confused by lack of symmetry here. Some fixtures are supposed to be executed, some imported. Are they still the same kind of things?

Reply:
this directory is a place for things that are run from tests. There is no other definition.